hack/vendor.sh

```diff
@@ -19,7 +19,15 @@ case $# in
 	;;
 # If user passed arguments to the script
 1)
-	eval "$(grep -E "^clone [^ ]+ $1" "$0")"
+	set +e
```

Comment:
Thanks for your review. @tianon 
Actually I like the latter one which uses a variable.
Updated the PR. PTAL